meeshkan/serve/admin/runner.py

```diff
@@ -9,18 +9,18 @@
     ScopeView,
     StorageView,
 )
-from meeshkan.serve.mock.rest import rest_middleware_manager
-from meeshkan.serve.mock.storage import storage_manager
+from meeshkan.serve.mock.rest import RestMiddlewareManager
+from meeshkan.serve.mock.storage.manager import StorageManager
```

Comment:
I use auto resolving in PyCharm and it can't insert relative imports. Checking it and modifying everything manually would be quite an overhead. I'm used to imports as something that is preserved automatically except for some corner cases.

Reply:
I think that would then be a reason to convert all imports to absolute ones in the repo in another PR.

Reply:
Ok, let's leave it as it is right now and discuss this refactoring later.